chaosazure/vmss/actions.py

```diff
@@ -81,7 +81,8 @@ def restart_vmss(filter: str = None,
 
 def stop_vmss(filter: str = None,
               configuration: Configuration = None,
-              secrets: Secrets = None):
+              secrets: Secrets = None,
+              instance_name: str = None):
```

Comment:
Merely a convention but we prefer keeping secrets as the last parameter. Usually it's a good idea the way you approached it because you didn't change the signature per-se but, because we call these parameters by name, their position doesn't matter. The fact we prefer config/secrets as last args is for our own consistency across our code base.